src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,62 @@
+package core.basesyntax;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DataConverter;
+import core.basesyntax.service.FileReader;
+import core.basesyntax.service.FileWriter;
+import core.basesyntax.service.ReportGenerator;
+import core.basesyntax.service.ShopService;
+import core.basesyntax.service.serviceimpl.DataConverterImpl;
+import core.basesyntax.service.serviceimpl.FileReaderImpl;
+import core.basesyntax.service.serviceimpl.FileWriterImpl;
+import core.basesyntax.service.serviceimpl.ReportGeneratorImpl;
+import core.basesyntax.service.serviceimpl.ShopServiceImpl;
+import core.basesyntax.stategy.OperationStrategy;
+import core.basesyntax.stategy.OperationStrategyImpl;
+import core.basesyntax.stategy.handler.BalanceOperation;
+import core.basesyntax.stategy.handler.OperationHandler;
+import core.basesyntax.stategy.handler.PurchaseOperation;
+import core.basesyntax.stategy.handler.ReturnOperation;
+import core.basesyntax.stategy.handler.SupplyOperation;
+import java.io.File;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    public static final String REPORT_TO_READ
+            = "src/main/resources/files/reportToRead.csv";
+    public static final String REPORT_TO_WRITE
+            = "src/main/resources/files/reportToWrite.csv";
+
+    public static void main(String[] arg) {
```

Comment:
There is an empty line at the beginning of the 'main' method implementation, which should be removed according to the checklist.